src/base/core/ports.c

```diff
@@ -122,7 +122,7 @@ void init_port_traceing(void)
 }
 
 #define TT_printf(p,f,v,m) ({ \
-  if (debug_level('T')) { \
+  if (debug_level('T') && test_bit(p, portlog_map)) {      \
```

Comment:
Logging full port access is quite useful.
Please make it, say,
`if (debug_level('T') && (test_bit(p, portlog_map) || debug_level('T') >=5))`